lib/ansible/modules/dnf5.py

```diff
@@ -496,7 +496,7 @@ def run(self):
             conf.config_file_path = self.conf_file
 
         try:
-            base.load_config_from_file()
+            base.load_config()
```

Comment:
Should we do a `try/except AttributeError` here preferring the new and falling back to the old?  I have no idea what versions of dnf5 are out there in the wild now.  I'm guessing the change as is should be fine, just asking, although since you did it this way, I guess I know your opinion.